scripts/prepare_sector_network.py

```diff
@@ -157,6 +157,113 @@ def add_generation(
         # set the "co2_emissions" of the carrier to 0, as emissions are accounted by link efficiency separately (efficiency to 'co2 atmosphere' bus)
         n.carriers.loc[carrier, "co2_emissions"] = 0
 
+    # add coal CC technologies
+    if options["coal"].get("CC", False):
+        coal_CC_techs = options["coal"].get("coal_CC_techs", list())
+
+        coal_CC_names = {
+            "coal CC-95": "Coal-95%-CCS",
+            "coal CC-99": "Coal-99%-CCS",
+            "coal IGCC CC-90": "Coal-IGCC-90%-CCS",
+        }

```

Comment:
Thanks for the insightful comments. Initially, I planned to provide full name, but I doubted to do, because they are typically too long. So I thought there is a some chance that user misspells something (of course we provide names as comments, but still), so I provided shorter versions which are explained in the config file. I agree having a dictionary inside `prepare_sector_network` is not nice. I will change to directly nice names, thus it will remove unnecessary mapping.

Reply:
Sounds absolutely reasonable. Agree that having a short name may be a good idea... I wonder if we can use `nice_names` argument in [the config](https://github.com/pypsa-meets-earth/pypsa-earth/blob/c12ce1add1ffbaafba4ff92dd2c821d00a6d73d5/config.default.yaml#L977-L989) to store those if they are needed? Then, in the code we could use those nice names if available and full names otherwise. Does it sound reasonable for you?